pkg/discovery/dtofactory/namespace_entity_dto_builder.go

```diff
@@ -40,6 +40,10 @@ func (builder *namespaceEntityDTOBuilder) BuildEntityDTOs() ([]*proto.EntityDTO,
 		}
 		entityDTOBuilder.SellsCommodities(commoditiesSold)
 
+		// Namespace entity cannot be provisioned or suspended by Turbonomic analysis
+		entityDTOBuilder.IsProvisionable(false)
+		entityDTOBuilder.IsSuspendable(false)
```

Comment:
What actions are we actually expecting on the workloadController ?

Reply:
Right, we don't want to generate any action on Namespaces and WorkloadControllers.